src/Forms/MainForm.cs

```diff
@@ -2290,7 +2308,77 @@ public static void forceGC()
         {
             GC.Collect();
             GC.WaitForPendingFinalizers();
-            SetProcessWorkingSetSize(System.Diagnostics.Process.GetCurrentProcess().Handle, -1, -1);
+            //SetProcessWorkingSetSize(System.Diagnostics.Process.GetCurrentProcess().Handle, -1, -1);
+        }
+
+        private void SaveSelectedObjectsToFile( int slot ) {
+            List<ObjectData> selectedObjects = new List<ObjectData>();
+
+            if( Globals.isMultiSelected ) {
+                if(
+                    Globals.isMultiSelectedFromMultipleLists ||
+                    Globals.multi_selected_nodes[0].Count < 1
+                ) return;
+
+                foreach( int index in Globals.multi_selected_nodes[0] ) {
+                    selectedObjects.Add(level.getCurrentArea().Objects[index].Data);
+                }
+            } else {
+                if ( Globals.list_selected != 0 || Globals.item_selected < 0 ) return;
+
+                selectedObjects.Add(level.getCurrentArea().Objects[Globals.item_selected].Data);
+            }
+
+            using( StreamWriter tempFile = new StreamWriter( "./data/profiles/default/saved-objects-" + slot.ToString() + ".json" ) ) {
+                tempFile.WriteLine(JsonConvert.SerializeObject(selectedObjects));
+            }
```

Comment:
Is it really required to save data between launches?

Reply:
It's useful for copying objects to/from a backup ROM or just to make a backup file of object placements.

One improvement might be to store the data in one place instead of basing it off of the working directory like Quad64 currently does, but I did it this way for now since this is where it normally stores it data.